posts/models.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+
+
+# Create your models here.
+class Post(models.Model):
+    title = models.CharField(max_length=20)
+    content = models.TextField()
+    created = models.DateTimeField(auto_now=True)
+    modified = models.DateTimeField(auto_now_add=True)
```

Comment:
To jest bardzo częsty pattern używany w modelach, więc powszechnym rozwiązaniem jest używanie TimeStampModel z appki https://github.com/django-extensions/django-extensions, spróbuj dodać ją do projektu i zobaczyć jak działa dziedziczenie po TimeStampModel (abstract=True)

Reply:
Post(TimestampModel)